### PR TITLE
End of Year: Implement design notes for the listening time

### PR DESCRIPTION
**Pull request by emilylaguna**
| 📘 Project: #376 | 🛫 Depends on: #557 |
|:---:|:---:|

This implements the following notes:
- Always show the full time unit string
   - This changes `1 hour 2 min` to `1 hour 2 minutes`
- Adjusts the font sizes / weights to match the designs
- Adjusts the line heights
- Adjusts the spacing between the diagonal stack and the labels to be inline with the designs
- I also tried adjusting the centering of the stack for larger devices, but it wasn't working on so I skipped it. 

### Design vs Implementation Comparison
<p align="center"></p>

### Devices
| iPhone SE - 14.1 | iPhone 13 | iPhone 14 Pro Max |
|:---:|:---:|:---:|
||||

## To test

1. Launch the app
2. Tap on the Profile Tab > End of Year Card
3. Tap to the second screen
4. ✅ Verify the view looks like the designs and looks good on all screen sizes

## Checklist

- [ ] I have considered if this change warrants user-facing release notes and have added them to `CHANGELOG.md` if necessary.
- [ ] I have considered adding unit tests for my changes.
- [ ] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.

### CODE REVIEW

File: Modules/Utils/Sources/PocketCastsUtils/Extensions/Double+formatTime.swift [hunk 1]
```diff
@@ -5,6 +5,17 @@ extension Double {
     ///
     /// Eg.: 5400 will return "1 hour 30 min"
     public var localizedTimeDescription: String? {
+        calculateDescription(useFullUnits: false)
+    }
+
+    /// Returns a localized time description using the full units for all description
+    ///
+    /// Eg.: 88479 will return "1 day 34 minutes 29 seconds"
+    public var localizedTimeDescriptionFullUnits: String? {
```
Comment (emilylaguna) [inline Modules/Utils/Sources/PocketCastsUtils/Extensions/Double+formatTime.swift:14]: This code is actually reverted in the next PR.

**Review by leandroalonso — COMMENTED** [review]
Tested and it works as described. There are two issue though, one I think we can fix in another PR the other might be better to tackle here:

### Shadow

The podcast shadow is super intense now, almost feels like a border. If you look at the designs and how it was before it's more subtle:



(The same happens on top 5 podcasts)

### Login and EoY prompt

This one can be tackled in another PR, but when logging in within the new flow the EoY modal is not shown (because it relied on the login always starting on the Profile screen).

**Review by leandroalonso — APPROVED** [review]
Given the shadow is fixed on a subsequent PR I'm approving this one.

**Comment by emilylaguna** [comment]
> This one can be tackled in another PR, but when logging in within the new flow the EoY modal is not shown (because it relied on the login always starting on the Profile screen).

I have a fix for this, will make a new PR for it.

**Pull request merged**